code/modules/mob/living/carbon/human/life.dm

```diff
@@ -35,6 +35,8 @@
 		//heart attack stuff
 		handle_heart()
 
+		handle_alcohol() //Alcohol
+

```

Comment:
I'm not a big fan of having every human call handle_alcohol() every tick when they will only have actual alcohol in their reagent like 1% of the time.
I made a 2k lines change PR simply to remove 2 proc calls in Life() so I'm not a big fan of adding new proc calls to Life().